unidist/core/backends/mpi/core/common.py

```diff
@@ -258,50 +258,60 @@ def get_op_name(op):
     return op_name
 
 
-class MasterDataID(DataID):
+class MpiDataID(DataID):
     """
-    Class for tracking data IDs of the main process.
+    Class for tracking data IDs of MPI processes.
 
-    Class extends ``unidist.core.backends.common.data_id.DataID`` functionality with a garbage collection.
+    The class extends ``unidist.core.backends.common.data_id.DataID`` functionality,
+    ensuring the uniqueness of the object to work correctly with MPI communication.
 
     Parameters
     ----------
-    id_value : int
-        An integer value, generated by executor process.
-    garbage_collector : unidist.core.backends.mpi.core.executor.GarbageCollector
-        A reference to the garbage collector instance.
+    owner_rank : int
+        The rank of the process that owns the data.
+    data_number : int
+        Unique data number for the owner process.
```

Comment:
```suggestion
        Unique data number for the owner process.
    gc : unidist.core.backends.mpi.core.executor.GarbageCollector or None
        Local garbage collector reference.
        The actual object is for the data id owner, otherwise, ``None``.
```